hack/pr-infra-deployments.sh

```diff
@@ -49,21 +49,18 @@ if [ -n "${GITHUB_ACTIONS:-}" ]; then
   trap 'rm -rf "${WORKDIR}" "${HOME}/.ssh/id_rsa"' EXIT
   export GITHUB_USER="$GITHUB_ACTOR"
 fi
-git checkout -b ec-policy-update --track upstream/main
 
-POLICY_SOURCE_URL=oci::https://${RELEASE_BUNDLE_REF}
-DATA_SOURCE_URL=oci::https://${DATA_BUNDLE_REF}
-# These would work too but we prefer the oci conftest bundle sources over direct github urls
-#POLICY_SOURCE_URL=git::https://github.com/hacbs-contract/ec-policies.git//policy?ref=${GIT_REF}
-#DATA_SOURCE_URL=git::https://github.com/hacbs-contract/ec-policies.git//data?ref=${GIT_REF}
+BRANCH_NAME=$REPO-update
+git checkout -b ${BRANCH_NAME} --track upstream/main
+
+SOURCE_URL=oci::https://${BUNDLE_REF}
 
 # replacements
-yq e -i '.configMapGenerator[] |= select(.name == "ec-defaults").literals[] |= select(. == "ec_policy_source=*") = "ec_policy_source='"${POLICY_SOURCE_URL}"'"' components/enterprise-contract/kustomization.yaml
-yq e -i '.configMapGenerator[] |= select(.name == "ec-defaults").literals[] |= select(. == "ec_data_source=*"  ) = "ec_data_source='"${DATA_SOURCE_URL}"'"'     components/enterprise-contract/kustomization.yaml
+yq e -i '.configMapGenerator[] |= select(.name == "ec-defaults").literals[] |= select(. == "'"${CM_KEY_NAME}"'"ec_policy_source=*") = "ec_policy_source='"${SOURCE_URL}"'"' components/enterprise-contract/kustomization.yaml
```

Comment:
fyi, it might simplify a bit to read the env var via `env(CM_KEY_NAME)`, see [docs](https://mikefarah.gitbook.io/yq/operators/env-variable-operators#read-string-environment-variable).  I believe you'd have to export the env var first though.

Reply:
Interesting idea, but I think I'll leave it alone for now.